src/nspawn/nspawn-stub-pid1.c

```diff
@@ -18,9 +18,9 @@
 ***/
 
 #include <sys/reboot.h>
-#include <sys/unistd.h>
 #include <sys/wait.h>
 #include <sys/prctl.h>
+#include <unistd.h>
```

Comment:
this one looks like a real bug fix applying to glibc systems too, this one definitely should go in